json-rpc/src/stream_rpc/logging.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use diem_logger::Schema;
+
+#[derive(Schema)]
+pub struct HttpRequestLog<'a> {
+    #[schema(display)]
+    pub remote_addr: Option<std::net::SocketAddr>,
+    pub path: &'a str,
+    pub status: u16,
+    pub referer: Option<&'a str>,
+    pub user_agent: &'a str,
+    pub forwarded: Option<&'a str>,
+}
+
+#[derive(Schema)]
+pub struct ClientConnectionLog<'a> {
+    pub transport: &'static str,
+    pub remote_addr: Option<&'a str>,
+    pub client_id: Option<u64>,
+    pub user_agent: Option<&'a str>,
+    pub forwarded: Option<&'a str>,
+    pub rpc_method: Option<&'static str>,
+}
```

Comment:
I'll leave it up to you, note, that you could potentially have a StreamRpcLog that has an action field, where the field is defined by

```
enum StreamRpcAction {
    client_connection: ClientConnectionLog,
    http_request: HttpRequestLog,
}
```

thus allowing a convergence, don't recall if this would be flattened or if we would need to flatten it. @bmwill / @gregnazario

Reply:
The schema would be the lowest level under `data.field_name`. Since there are no complex objects being serialized, they will all be flattened.